src/components/navbar.component.tsx

```diff
@@ -56,6 +65,43 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
     };
   }, []);
 
+  let [searchBarIsVisible, setSearchBarIsVisible] = React.useState<any>(false);
+
+  let [closeSearchBar, setCloseSearchBar] = React.useState<any>(false);
+
+  let setSearchBar = () => {
+    setSearchBarIsVisible((searchBarIsVisible = true));
+    if (searchBarIsVisible === true) {
+      document
+        .getElementById("searchBar")
+        .classList.add(styles.searchOverlayOn);
+
+      document.getElementById("searchIcon").classList.add(styles.search);
+    } else {
+      document
+        .getElementById("searchBar")
+        .classList.add(styles.searchOverlayOn);
+
+      document
+        .getElementById("searchIcon")
+        .classList.add(styles.searchIconOverlayed.display.none);
+    }
+  };
+
+  let closeSerachBarRef = React.useRef(null);
+
+  let handleSearcbBarClose = e => {
```

Comment:
```suggestion
  const handleSearcbBarClose = e => {
```
this variable isn't going to be reassigned best to use `const`